app/views/YourSelfAssessmentCharges.scala.html

```diff
@@ -285,8 +308,17 @@
                 target = Some("_blank"),
                 outerMessage = ".")
         </p>
-    }
 
+    @if(viewModel.claimToAdjustViewModel.adjustPaymentsOnAccountFSEnabled) {
+        <p class="govuk-body" id="adjust-payments-on-account">
+            @link(
+                link = controllers.claimToAdjustPoa.routes.AmendablePoaController.show(user.isAgent()).url,
+                messageKey = getMessage("adjust-poa", "2024", "2025"),
```

Comment:
not sure if this is a good idea for message to be resolved by specific tax years / calendar years